node/trieIterators/factory/delegatedListHandlerFactory_test.go

```diff
@@ -8,25 +8,33 @@ import (
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-go/node/mock"
 	"github.com/multiversx/mx-chain-go/node/trieIterators"
+	trieIteratorsFactory "github.com/multiversx/mx-chain-go/node/trieIterators/factory"
 	"github.com/multiversx/mx-chain-go/testscommon"
 	stateMock "github.com/multiversx/mx-chain-go/testscommon/state"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestCreateDelegatedListHandlerHandler_Disabled(t *testing.T) {
+func TestNewDelegatedListHandlerFactory(t *testing.T) {
```

Comment:
Please also rename the unit test names after the components are renamed.

! Applies globally in this file